fetchy/fetchyboi.ts

```diff
@@ -29,34 +31,49 @@ const sendDataUpdatedNotification = async () => {
   );
 };
 
-const areObjectsDifferent = (a: object, b: object) =>
-  Object.keys(diff(a, b)).length !== 0;
+const areObjectsDifferent = (a: Object, b: Object) =>
```

Comment:
I think actually lowercase object is possibly better here - I sure don't know haha
```suggestion
const areObjectsDifferent = (a: object, b: object) =>
```

Reply:
I have no idea haha, I think your commit made them upper case so I left it like that 😂